polarCoordinateTraversalTest.m

```diff
@@ -43,18 +43,235 @@ function testRayEntersCircleAndGoesThroughOrigin(testCase)
     
     [rVoxels, thetaVoxels] = polarCoordinateTraversal(min_bound, max_bound, ray_origin, ray_direction, ...
         circle_center, circle_max_radius, num_radial_sections, num_angular_sections, t_begin, t_end, verbose);
-    expected_rVoxels     = [1,2,2,2,1];
-    expected_thetaVoxels = [2,2,1,0,0];
```

Comment:
This was incorrect before you changed it?

Reply:
![Screen Shot 2020-02-27 at 6 02 31 PM](https://user-images.githubusercontent.com/37983775/75494960-64853880-598b-11ea-8325-fbee25966b7b.png)

Reply:
Here I am seeing:
rVoxels: [1,2,2,2,1]
thetaVoxels: [2,2,1,0,0]

Reply:
Can you explain what you changed for my understanding?

Reply:
Resolved in Slack.